locales/templates/app.pot

```diff
@@ -259,6 +255,20 @@ msgid ""
 "sign you out of websites and delete web content that is stored offline."
 msgstr ""
 
+msgctxt "settings_cookies_confirm"
+msgid "Clear now"
+msgstr ""
+
+msgctxt "preference_mozilla_telemetry2"
+msgid "Send usage data"
```

Comment:
Please provide a short localization comment to give some context

Reply:
This one has already been translated, it looks like it was just moved.  I'll add a comment, but we shouldn't retranslate it.

Reply:
Cool thanks! Looks like that fell through the cracks at one point in my review then :)
I looked and this still appears translated in other languages, so we should be good. Thank you!